benchmarks/sha3.cc

```diff
@@ -99,4 +100,29 @@ Hacl_Sha3_shake256(benchmark::State& state)
 
 BENCHMARK(Hacl_Sha3_shake256);
 
+static void
+OpenSSL_Sha3_256_Streaming(benchmark::State& state)
+{
+  size_t chunk_len = 135;
+
+  while (state.KeepRunning()) {
+    // Init
+    EVP_MD_CTX* ctx = EVP_MD_CTX_new();
+    EVP_DigestInit_ex(ctx, EVP_sha3_256(), NULL);
```

Comment:
Question: OpenSSL newby here: Can we use `EVP_DigestInit` instead of `EVP_DigestInit_ex`? Seems like we don't need `_ex` anyway because we use `NULL`.